include/backend/wayland.h

```diff
@@ -108,6 +108,7 @@ struct wlr_wl_pointer {
 struct wlr_wl_seat {
 	struct wl_seat *wl_seat;
 
+	struct wl_list link;
```

Comment:
Ditto:

```c
struct wl_list link; // wlr_wl_backend.seats
```

Reply:
I see benefit of previous one because `wl_list` doesn't suggest type of list elements. But I'm not fully sure about practice of referencing all the places where we reference this list type.

I don't insist on changing practice in this project. Just suggest to re-consider what we get out of it and how it works in case if we have zero (e.g. callbacks binding) or more than one fields where we have list of list-capable structs.

Reply:
The comments have proven useful when new lists are added to a struct and when an item can be added to multiple lists.